docs/tutorials/07-filters.md

```diff
@@ -181,3 +181,79 @@ An example of time filter is available under `examples/sensors`.
          ┃           stop        start       ┃       stop            start       ┃
         ─┸───────────○───────────●───────────┸───────────○───────────●───────────┸─
         ─── OK ──────╯           ╰───────── OK ──────────╯           ╰────── OK ───
+
+## Script Filter
+
+Script filter evaluates a Lua script to determine the validity of the event.
+
+> Available after v1.5.5
```

Comment:
Maybe better something like `since v1.5.6` because using "after" seems already included in v1.5.5 which is not...